### PR TITLE
Use xdelta3 instead of xdelta (which is being dropped/obsoleted)

### PR DESCRIPTION
**Pull request by DimStar77**

### CODE REVIEW

File: tests/console/sshfs.pm [hunk 1]
```diff
@@ -27,8 +27,8 @@ sub run() {
     send_key "ret";
     assert_screen 'sshfs-accepted';
     script_run('cd mnt/tmp');
-    assert_script_run("zypper -n in xdelta");
-    assert_script_run("rpm -e xdelta");
+    assert_script_run("zypper -n in xdelta3");
+    assert_script_run("rpm -e xdelta3");
```
Comment (DimStar77) [inline tests/console/sshfs.pm]: Grmbl.. sshfs also runs on SLE12 anywhere? there we miss xdelta3 it seems :(

File: tests/console/zypper_in.pm [hunk 2]
```diff
@@ -23,7 +23,7 @@ sub run() {
     my $pkgname = get_var('PACKAGETOINSTALL');
     if (!$pkgname) {
         $pkgname = 'x3270'  if check_var('DISTRI', 'sle');
-        $pkgname = 'xdelta' if check_var('DISTRI', 'opensuse');
+        $pkgname = 'xdelta3' if check_var('DISTRI', 'opensuse');
```
Comment (DimStar77) [inline tests/console/zypper_in.pm]: This one is safe, as xdelta3 has been in openSUSE since version 12.2

File: tests/console/sshfs.pm (hunk 1 above)
Comment (okurz) [inline tests/console/sshfs.pm]: `select * from job_modules where (name='console-sshfs') limit 10;` did not return anything from the osd database and also sshfs is only mentioned in `products/opensuse/main.pm` so we are safe :-)

**Comment by okurz** [comment]
https://travis-ci.org/os-autoinst/os-autoinst-distri-opensuse/builds/245890087?utm_source=github_status&utm_medium=notification says you need to update maintainer and summary of sshfs. While you are at it: Do you think it would be useful to trigger the same module for SLE as well? I don't see a reason why not so but that should not block your PR here.

**Comment by DimStar77** [comment]
Actually in an earlier version (which had formatting issues) I was wondering if sshfs is being used on SLE too - as then this PR would cause an issue, due to xdelta3 not being available on SLE (I'm sure nobody knows why)

**Comment by okurz** [comment]
FYI reg. SLE: We do not have sshfs within SLE unless using packagehub

**Pull request merged**